README.md

```diff
@@ -706,6 +706,19 @@ read_by_id = client.read_note(written.note_id)
 print(read_by_id.data['lyrics'])
 ```
 
+## Identity
+Currently the Python SDK has limited implementation of the Tozny Identity primatives. Currently the functionality exists to login an existing Identity created in the Tozny Identity console. Login from the Python can be performed by calling the static identity_login method in the Identity class. The method requires the user_name, password, realm name and app name. An instance of the Identity class is returned, which includes the OAuth tokens necessary to interact with the Tozny Identity service.
+
+Currently the Python SDK does not support Multi Factor Authentication (MFA) or Brokered login. 
+app_name has to be account, we do not currently support any other app_name
```

Comment:
If an application in the realm management dashboard to support API style login does it still fail? I would expect other applications to work with the most basic flow.

Reply:
Updates made per our conversation